src/components/workspace/settings/AttentionSign.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div class="attention">
+    <div class="attention__sign">
+      !
+    </div>
+  </div>
+</template>
+
+<script>
+import Vue from 'vue';
+
+export default Vue.extend({
+  name: 'AttentionSign',
+});
+
+</script>
+
+<style lang="postcss">
```

Comment:
Не обязательно указывать lang
```suggestion
<style>
```